codegenerator/cli/npm/envio/src/sources/HyperSyncClient.res

```diff
@@ -307,6 +308,30 @@ module ResponseTypes = {
     storageKeys?: array<string>,
   }
 
+  let accessListSchema = S.object(s => {
+    address: ?s.field("address", S.option(Address.schema)),
+    storageKeys: ?s.field("storageKeys", S.option(S.array(S.string))),
+  })
+
+  // TODO: update some of these fields to be integers rather than strings.
```

Comment:
This should be exposed via `@genType` to make it work in TS

Reply:
Also, it would be nice to add a test for the types. Here's the reference https://github.com/enviodev/hyperindex/blob/51909ebf6cf1b750a64433031cef25b27b87fc38/scenarios/test_codegen/src/EventHandlers.ts#L150-L170

Reply:
Thanks, works!
![image](https://github.com/user-attachments/assets/8f3599b1-faf3-451c-ae4a-70eb122fc3ef)
Will update properly after meeting.